modules/post-installation-adding-nutanix-failure-domains-control-planes.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-control-planes_{context}"]
+= Distributing control planes across failure domains
+
+You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource (CR).
+
+.Prerequisites
+
+* You have added the failure domains to the cluster's Infrastructure custom resource definition (CRD).
```

Comment:
Should change to:
* You have added the failure domains configuration to the cluster's Infrastructure custom resource (CR).

Reply:
Updated.